From: Joel Savitz <jsavitz@redhat.com>
Subject: selftests/vm: remove extraneous echo
Date: Wed, 16 Nov 2022 20:06:32 -0400

There is an extra echo statement in run_test(). Remove it.

Link: https://lkml.kernel.org/r/20221117000632.254027-1-jsavitz@redhat.com
Fixes: 98b3add737b4 ("selftests/vm: enable running select groups of tests")
Signed-off-by: Joel Savitz <jsavitz@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/vm/run_vmtests.sh |    1 -
 1 file changed, 1 deletion(-)

--- a/tools/testing/selftests/vm/run_vmtests.sh~selftests-vm-enable-running-select-groups-of-tests-fix-2
+++ a/tools/testing/selftests/vm/run_vmtests.sh
@@ -142,7 +142,6 @@ echo "$ARCH64STR" | grep "$ARCH" &>/dev/
 # Usage: run_test [test binary] [arbitrary test arguments...]
 run_test() {
 	if test_selected ${CATEGORY}; then
-		echo "running: $1"
 		local title="running $*"
 		local sep=$(echo -n "$title" | tr "[:graph:][:space:]" -)
 		printf "%s\n%s\n%s\n" "$sep" "$title" "$sep"
_
